pkgs/applications/video/vdr/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, fetchurl, fontconfig, libjpeg, libcap, freetype, fribidi, pkgconfig
+, gettext, ncurses, systemd, perl
+, enableSystemd ? true
+, enableBidi ? true
+}:
+let
+
+  version = "2.4.0";
+
+  name = "vdr-${version}";
+
+  mkPlugin = name: stdenv.mkDerivation {
+    name = "vdr-${name}-${version}";
+    inherit (vdr) src;
+    buildInputs = [ gettext vdr ];
+    nativeBuildInputs = [ pkgconfig ];
+    preConfigure = "cd PLUGINS/src/${name}";
+    installFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
+  };
+
+  vdr = stdenv.mkDerivation {
+
+    inherit name;
+
+    src = fetchurl {
+      url = "ftp://ftp.tvdr.de/vdr/${name}.tar.bz2";
+      sha256 = "1klcgy9kr7n6z8d2c77j63bl8hvhx5qnqppg73f77004hzz4kbwk";
+    };
+
+    enableParallelBuilding = true;
+
+    postPatch = "substituteInPlace Makefile --replace libsystemd-daemon libsystemd";
+
+    buildInputs = [ fontconfig libjpeg libcap freetype perl ]
+      ++ stdenv.lib.optional enableSystemd systemd
+      ++ stdenv.lib.optional enableBidi fribidi;
+
+    buildFlags = [ "vdr" "i18n" ]
+      ++ stdenv.lib.optional enableSystemd "SDNOTIFY=1"
+      ++ stdenv.lib.optional enableBidi "BIDI=1";
+
+    propagatedBuildInputs = [ pkgconfig gettext ];
```

Comment:
Should this not be in nativeBuildInputs?

Reply:
Every vdr plugin need to have the same build tools than vdr itself. If will you propagatedNavtiveBuildInputs. If you want the plugin descriptions more explicit I can change it to no propagated.

Reply:
The problem is that it also propagate to every other package that uses vdr somehow. You can abstract it way in a variable or so or use `vdr.propagatedBuildInputs` explicit.